src/Module/Admin/Summary.php

```diff
@@ -55,6 +55,16 @@ public static function content(array $parameters = [])
 			$warningtext[] = DI::l10n()->t('Your DB still runs with InnoDB tables in the Antelope file format. You should change the file format to Barracuda. Friendica is using features that are not provided by the Antelope format. See <a href="%s">here</a> for a guide that may be helpful converting the table engines. You may also use the command <tt>php bin/console.php dbstructure toinnodb</tt> of your Friendica installation for an automatic conversion.<br />', 'https://dev.mysql.com/doc/refman/5.7/en/innodb-file-format.html');
 		}
 
+		// Avoid the database error 1615 "Prepared statement needs to be re-prepared", see https://github.com/friendica/friendica/issues/8550
+		$table_definition_cache = DBA::getVariable('table_definition_cache');
+		$table_open_cache = DBA::getVariable('table_open_cache');
+		if (!empty($table_definition_cache) && !empty($table_open_cache)) {
+			$suggested_definition_cache = max(400, round($table_open_cache / 2, 1));
```

Comment:
How did you decide on this value?

Reply:
No, they are described in the linked documentation.

Reply:
In the linked documentation, it only says that the default value is `400 + table_open_cache /2` which isn't the calculation you made here. It doesn't say it's the suggested value either.

Reply:
The value shouldn't be lower than that.

Reply:
The minimum value for `table_definition_cache` is `400`, it has nothing to do with the default value which takes `table_open_cache` into account. If you make the following change you don't even have to use `max()`.
```suggestion
			$suggested_definition_cache = 400 + round($table_open_cache / 2, 1);
```

Reply:
No, according to the documentation they are using the `max` functionality as well. So it doesn't add, 400 is just the minimum value.

Reply:
Can you please show me where? At this point I feel like we're looking at different pages. Here's my reference for what I say above:
<img width="1070" alt="Screen Shot 2020-04-29 at 15 35 59" src="https://user-images.githubusercontent.com/925415/80638971-5016f800-8a2f-11ea-8f0c-bb932f0b4b7f.png">
It says `minimum of 400` (included in both our codes), `capped at 2000` (missing in both our codes), and it clearly has a plus sign (`+`) in the default value formula. Please note that it is a default value, not a suggested one either.